jina/clients/__init__.py

```diff
@@ -98,7 +98,7 @@ def index(
         on_error: CallbackFnType = None,
         on_always: CallbackFnType = None,
         **kwargs
-    ) -> None:
+    ) -> Awaitable:
```

Comment:
@Kelton8Z Is `Awaitalbe` the correct type? That is the sync client - not the async variation.

Reply:
is index a sync client in the sense that the flow needs to finish indexing before moving on to query or other process?

in that case i also need to change the code within the index function ?